tests/jballerina-unit-test/src/test/resources/test-src/record/readonly_record_fields_negative.bal

```diff
@@ -284,3 +284,20 @@ type RecordWithReadOnlyFields record {|
 
 RecordWithReadOnlyFields & readonly r1 = {x: []};
 readonly & RecordWithReadOnlyFields & readonly r2 = 1;
+
+type openNeverRecord record {
```

Comment:
Fix names here as well.

Reply:
@ushirask seems you have missed updating this file.

Reply:
Updated.